md_core/mdpy/fixref/start.py

```
@@ -51,7 +52,7 @@ def main():
             thenumbers[1] = int(value)
         # ---
         if arg == '-file':
-            text = open(f'{dir2}/public_html/find/{value.strip()}', "r", 'utf8').read()
+            text = open(f'{public_html_dir}/find/{value.strip()}', "r", 'utf8').read()
```

Ruff (0.8.0)

Use a context manager for opening files

(SIM115)